web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php

```diff
@@ -76,6 +84,16 @@ class ProfilesBlock extends BlockBase implements ContainerFactoryPluginInterface
   protected $pageNumber;
 
   /**
+   * Quarantined filter state.
+   *
+   * Positive if the list of profiles should be filtered by profiles with an
+   * active quarantine.
+   *
+   * @var int $quarantinedFilter
```

Comment:
How about making this a boolean instead of an integer?

That seems to be more appropriate for what the filter actually does.